### PR TITLE
Add vulnerability ui components and initial summary page

### PR DESCRIPTION
**Pull request by mmmmillar**
**What**  
Add vulnerability ui components and initial summary page

**Why**  
To get the UI component structure and page sorted before beginning work on saving the snapshot

Screenshot 2020-06-18 at 19 06 41

### CODE REVIEW

File: components/Feature/VulnerabilitiesGrid/index.js [hunk 1]
```diff
@@ -0,0 +1,58 @@
...
+      assets: [{ text: 'a1' }],
+      vulnerabilities: [{ text: 'v1' }, { text: 'v2' }]
+    },
+    {
+      id: 'group2',
+      name: 'this is another group',
+      assets: [{ text: 'a1' }],
+      vulnerabilities: [{ text: 'v1' }, { text: 'v2' }]
+    }
+  ];
```
Comment (kosiakkatrina) [inline components/Feature/VulnerabilitiesGrid/index.js]: Would it be worth extracting these into like a json file? (I assume it'll be hardcoded in the future?)

File: components/Form/CheckboxList/index.test.js [hunk 2]
```diff
@@ -0,0 +1,9 @@
+import { render } from '@testing-library/react';
+import CheckboxList from './index';
+
+describe('Checkbox', () => {
```
Comment (t-davies) [inline components/Form/CheckboxList/index.test.js]:
```suggestion
describe('CheckboxList', () => {
```

**Review by t-davies — APPROVED** [review]
Looks pretty nice and generic!

File: components/Feature/VulnerabilitiesGrid/index.js (hunk 1 above)
Comment (mmmmillar) [reply components/Feature/VulnerabilitiesGrid/index.js]: done!

**Pull request merged**